packages/table/test/editableCellTests.tsx

```diff
@@ -126,4 +126,16 @@ describe("<EditableCell>", () => {
         elem.find("input").simulate("blur");
         expect(onChangeSpy.firstCall.args).to.deep.equal([CHANGED_VALUE, ROW_INDEX, COLUMN_INDEX]);
     });
+
+    it("defaults to no wrapText", () => {
+        const elem = mount(<EditableCell />);
+
+        expect(elem.find(`.${Classes.TABLE_NO_WRAP_TEXT} div`).length).to.equal(2);
+    });
+
+    it("wraps text when wrapsText is true", () => {
+        const elem = mount(<EditableCell wrapText={true} />);
+
+        expect(elem.find(`.${Classes.TABLE_NO_WRAP_TEXT} div`).length).to.equal(0);
```

Comment:
This is an inscrutable selector. Why the child div? Is it necessary? Can you come up with a more legible assertion? Maybe just the length of children on TABLE_NO_WRAP_TEXT?

Reply:
Thank you @giladgray.   You are correct, the div is not necessary.   Added children().length.  Is this what you  wanted?